apps/exploits/api.py

```diff
@@ -38,6 +40,8 @@ class ExploitsCollect(APIView):
     **NOTE:** Currently for CISA data only, which is very small and collection is fast.
     """
 
+    http_method_names = get_valid_http_methods(ModelViewSet)
```

Comment:
Was this consulted with @skontar ?

Reply:
I have talked with @skontar and he believes it should stay and allow user to start the collector in read mode. I have removed the changes and kept the endpoint.